var/spack/repos/builtin/packages/py-grpcio/package.py

```diff
@@ -36,16 +49,38 @@ class PyGrpcio(PythonPackage):
     version("1.28.1", sha256="cbc322c5d5615e67c2a15be631f64e6c2bab8c12505bc7c150948abdaa0bdbac")
     version("1.27.2", sha256="5ae532b93cf9ce5a2a549b74a2c35e3b690b171ece9358519b3039c7b84c887e")
     version("1.25.0", sha256="c948c034d8997526011960db54f512756fb0b4be1b81140a15b4ef094c6594a4")
-    version("1.16.0", sha256="d99db0b39b490d2469a8ef74197d5f211fa740fc9581dccecbb76c56d080fce1")
+    version(
+        "1.16.0",
+        sha256="d99db0b39b490d2469a8ef74197d5f211fa740fc9581dccecbb76c56d080fce1",
```

Comment:
Re-checked version sha256s and am now getting the following for this package:

```suggestion
        sha256="0cc5f2d3ee21c642d8982f197c83053fd3a8cbcd6a60240d8c87c6c256b10d57",
```

Haven't looked at why it changed.

Reply:
Bizarre! I'll recheck on my side as well.